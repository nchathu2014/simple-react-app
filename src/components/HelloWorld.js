import React,{PropTypes} from 'react';

const HelloWorld = () => {
    return (
        <div className='hello-world'>
            <h1>ReactJS Seed</h1>
            <p>Hi new p</p>
            <p>Welcome to ReactJS Seed</p>
        </div>
    )
};

HelloWorld.propTypes={

};

export default HelloWorld;