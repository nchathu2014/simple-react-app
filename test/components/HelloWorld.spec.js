import React from 'react';
import expect from 'expect';
import HelloWorld from './../../src/components/HelloWorld';

//Testing basic component rendering
test('render HelloWorld component', () => {
    const wrapper = shallow(<HelloWorld/>);
    expect(wrapper).toMatchSnapshot();
});