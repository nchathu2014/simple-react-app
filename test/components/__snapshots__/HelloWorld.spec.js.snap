// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render HelloWorld component 1`] = `
<div
  className="hello-world"
>
  <h1>
    ReactJS Seed
  </h1>
  <p>
    Hi new p
  </p>
  <p>
    Welcome to ReactJS Seed
  </p>
</div>
`;
